contracts/Bridge.sol

```diff
@@ -317,9 +320,9 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         uint64 depositNonce = ++_depositCounts[destinationChainID];
 
         IDepositExecute depositHandler = IDepositExecute(handler);
-        depositHandler.deposit(resourceID, destinationChainID, depositNonce, msg.sender, data);
+        bytes memory metaDataHandlerResponse = depositHandler.deposit(resourceID, msg.sender, data);
 
-        emit Deposit(destinationChainID, resourceID, depositNonce);
+        emit Deposit(destinationChainID, resourceID, depositNonce, msg.sender, data, metaDataHandlerResponse);
```

Comment:
Could you please add some docs for this new metaData

Reply:
Added.
Please check.